docs/how-to/change-log-level.rst

```diff
@@ -0,0 +1,34 @@
+===============================
+Change log level in MicroCeph
+===============================
```

Comment:
let's just say: "Changing the log level"
and ensure that the underlines/overlines are the same length

Reply:
done.

Reply:
just "Changing the log level"

as in all other page titles, it's understood to be MicroCeph